help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -2052,6 +2052,9 @@ Command sequences should also be enclose in "strong" formatting.    jerryg2003 2
         the <a href="#TrackBlockingOrder">Track Blocking Order</a>.
       </p>
 
+      <p>If you want your locomotive set outs to appear after your car set outs select the
+        option "Print Loco Set Outs Last".</p>
```

Comment:
Is this feature already added in a previous PR? Maybe I'm missing something, but I don't see that change in this PR.

Reply:
Correct, updating help, previous PR added the loco set outs last.